addons/html_editor/static/tests/_helpers/editor.js

```diff
@@ -163,7 +165,19 @@ export async function testEditor(config) {
     editor.shared.history.stageSelection();
 
     if (config.props?.iframe) {
-        expect("iframe").toHaveCount(1);
+        const selection = editor.document.getSelection();
+        // If there is no selection, iframe count remains 1 on both mobile
+        // and desktop since the toolbar is not open.
+        // When a selection exists:
+        //   - On mobile: The toolbar remains open regardless of whether
+        //     selection is collapsed or not, so the iframe count is 2.
+        //   - On desktop: The toolbar opens only when selection is not
+        //     collapsed, resulting in 2 iframes; otherwise, it remains 1.
```

Comment:
Good comment, but you forgot to explain why there are 2 iframes when the toolbar is opened. 2 iframes because of the font size input that is inside its own iframe.